src/App.js

```diff
@@ -1,13 +1,25 @@
 import React from "react";
-
+import Header from "./Header";
 import Bookings from "./Bookings";
+import TouristInfoCards from "./TouristInfoCards";
+import Footer from "./Footer";
+import Locationinfo from "./data/locationinfo.json";
 import "./App.css";
 
+const address = [
+  "123 Fake Street, London, E1 4UD",
+  "hello@fakehotel.com",
+  "0123 456789"
+];
+const name = " CYF hotel";
+
 const App = () => {
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
+      <Header name={name} />
+      <TouristInfoCards data={Locationinfo} />
       <Bookings />
+      <Footer address={address} />
```

Comment:
"address" is a slightly strange name to use for this prop, both because it's not just one address, and because not everything in the array is an address. Can you think of a better name?

Reply:
I changed to contactInfos